### ISSUE TITLE
Relate EasyAccessibility schema to codes in Woordenboek Reizigerskenmerken

### ISSUE THREAD

**Issue by efel85** [efel85/TOMP-API#17]

**Comment by efel85** [comment]
Maybe EasyAccessibility should not be part of the type-of-asset query, as accessibility knowledge should be available at the MSP

**Comment by edwinvandenbelt** [comment]
I think it's now more generic: in the availability-check you can require these items. I also added a 'meta' dictionary to the type-of-asset, that could be used for this purpose.

**Comment by edwinvandenbelt** [comment]
In the availability-check there is also a requirements- field. In this field these 'kenmerken' can be put, to be use by the TO.
e.g. requirements: [ {"HR-01": "true", "HR-06": "Voetsteun steekt vooruit" } ]

**Comment by efel85** [comment]
Woordenboek Reizigerskenmerken to be added as an appendix to the Blueprint v1.1
